crates/astria-conductor/src/conductor.rs

```diff
@@ -119,11 +122,18 @@ impl Conductor {
         }
 
         if !cfg.execution_commit_level.is_firm_only() {
-            // todo!("set up stuff");
             let (shutdown_tx, shutdown_rx) = oneshot::channel();
             let (sync_done_tx, sync_done_rx) = oneshot::channel();
+
+            // the `sync_start_block_height` below is the height at which the sync
+            // of Conductor will start. This value is a combination of the
+            // initial sequencer block height, which is set in the config when
+            // the rollup is first created, plus the height of the most recently
+            // executed block on the rollup.
+            // sync_start_block_height =
+            //   initial sequencer block height + most recently executed block height
```

Comment:
I think we can simplify this description a bit. The sequencer reader starts sync at the height which is the next executable block. How that's derived is more of an executor concern.

Reply:
updated the comment